app/views/providers/check_benefits/index.html.erb

```diff
@@ -1,5 +1 @@
-<% if should_use_ccms? %>
-  <%= render 'use_ccms' %>
```

Comment:
By removing the redundant `use_ccms_arbiter` service class is the `_use_ccms.html.erb` partial also redundant?